Description: Try to fix Ticket #95
URL: https://sf.net/p/opendmarc/tickets/95/
Author: A. Schulze
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: opendmarc-1.3.1/libopendmarc/opendmarc_spf.c
===================================================================
--- opendmarc-1.3.1.orig/libopendmarc/opendmarc_spf.c
+++ opendmarc-1.3.1/libopendmarc/opendmarc_spf.c
@@ -113,6 +113,7 @@ opendmarc_spf2_specify_ip_address(SPF_CT
 	 * we don't care at this point if it is ipv6 or ipv4
 	 */
 	SPF_request_set_ipv4_str(spfctx->spf_request, ip_address);
+	SPF_request_set_ipv6_str(spfctx->spf_request, ip_address);
 	return 0;
 }
 
